### PR TITLE
Newsletters: add #19 (2018-10-30)

### PR DESCRIPTION
**Pull request by harding**
I'm terribly, terribly sorry this is so late.  :-(

Otherwise, hopefully it's ok.  I won't be around this evening to address feedback, but I will check first thing tomorrow morning (around 06:00 EDT) and make any requested changes then.

### CODE REVIEW

**Comment by jnewbery** [comment]
> I'm terribly, terribly sorry this is so late. :-(

No problem! Thanks as ever for writing it. I'll review it this evening.

@moneyball - are you still at work and feel like reviewing?

**Comment by moneyball** [comment]
I made a few minor changes - take a look at them to make sure they're correct.

I also read the entire newsletter and tested it locally. Well written! I do not have the technical depth and history to confirm all of the info in notable merges, but it all seems to make sense to me.

tACK

File: _posts/en/newsletters/2018-10-30-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+    output was always change---making coin tracking trivial).  Havar
+    then suggests an alternative deterministic ordering based on private
+    information that would be available to the test suite but not
+    exposed by production wallets, allowing developers who want to
+    thwart block chain analysis---but also have simple and fast
+    tests---to migrate away from BIP69.
+
+- **Overt ASICBoost support for S9 miners:** support for this
+  efficiency-improving feature was announced by both [Bitmain][bitmain oab]
+  and [Braiins][braiins oab] this week.  Both covert and overt ASICBoost
```
Comment (jnewbery) [inline _posts/en/newsletters/2018-10-30-newsletter.md]: I think this sentence could be clearer. I've made an attempt in my commit, but I'm not sure if it's any better. Let me know what you think.

We could also link directly to one of the following:
the ASICboost paper : http://www.math.rwth-aachen.de/~Timo.Hanke/AsicBoostWhitepaperrev5.pdf
stack exchange answer: https://bitcoin.stackexchange.com/questions/43579/how-does-asicboost-work
blog post by ASICboost author: https://bitslog.wordpress.com/2017/04/10/the-relation-between-segwit-and-asicboost-covert-and-overt/

**Comment by jnewbery** [comment]
@harding - I've pushed a suggested change in a commit. Feel free to squash it or drop it.

Otherwise looks great. ACK fbb4746caf0cb7eb173e60180268e13ab8fad6f1

@moneyball - your changes look good. Thanks.

**Comment by harding** [comment]
Thanks both of you!  @jnewbery your description of ASICBoost was *much* better than mine!

I made a final small tweak to the introduction in and rephrased the text about Pierre's blog posts given that the residency program had more days than he has posts.  For review, those changes are in commit  d91a415 on a preservation of the full branch.  Otherwise, I've squashed on this branch.

Thanks again!

**Comment by jnewbery** [comment]
Thanks Dave. I've pushed one final change.

The residency was over 5 days, but the guest speakers gave their presentations on Mon-Thurs (Friday was left for residents to finish their projects and give demos). I've reworded the section to make it clear that Pierre's articles summarize all of the presentations. Let me know what you think.

**Comment by jnewbery** [comment]
squashed and merged

**Pull request merged**